.ci/packaging.groovy

```diff
@@ -218,6 +244,22 @@ def release(){
   }
 }
 
+def triggerE2ETests(String suite, String channel) {
+  build(job: "../../e2e-tests/e2e-testing-mbp/${env.JOB_BASE_NAME}",
```

Comment:
@elastic/observablt-robots how do we build the relative path here? I tried with and without the dots, without success. See error here: https://beats-ci.elastic.co/job/Beats/job/packaging/job/PR-21100/6/console

Reply:
I guess thee JOB_BASE_NAME does not match any of the jobs in the MBP:
- https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-mbp/

It might work for branches but for testing this PR it might not work since they are different git projects.

Maybe you would like to use the target branch env variable (see `CHANGE_TARGET` in [here](https://beats-ci.elastic.co/pipeline-syntax/globals)).

You can use `e2e-tests/e2e-testing-mbp/${env.JOB_BASE_NAME}` instead `../../..`